<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiSettingsDialog</class>
 <widget class="QDialog" name="MidiSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="_midiOnBox">
       <property name="toolTip">
        <string>Use MIDI input to control LLP</string>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Enable MIDI control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_inputLabel">
       <property name="text">
        <string>MIDI Input</string>
       </property>
       <property name="buddy">
        <cstring>_inputSelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_inputSelector">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a MIDI input device</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_refreshButton">
       <property name="toolTip">
        <string>Refresh the list of MIDI inputs</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Images/llp.qrc">
         <normaloff>:/Images/Refresh</normaloff>:/Images/Refresh</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="_settingsTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Operation</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shortcut</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MIDI Setting</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="fileBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Open|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Images/llp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MidiSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MidiSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_midiOnBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>_inputSelector</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_midiOnBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>_refreshButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_midiOnBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>_inputLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
